tasks/main.yml

```diff
@@ -39,7 +38,7 @@
 - name: Configure mariadb
   notify: Restart MariaDB
   ansible.builtin.ini_file:
-    dest: /etc/my.cnf.d/opencast.cnf
+    dest: "{{ mariadb_configuration_file }}"
```

Comment:
Multiple places to comment on this, but let me make the comment here once.

Why set this to different places on Debian and Red Hat? The previous location should be fine. Setting this to something else will just cause confusion for people upgrading from an older version of this role and I don't see any benefit.